cmd/repl/repl.go

```diff
@@ -105,15 +105,18 @@ func flagPassed(name ...string) bool {
 	return found
 }
 
-func runStmt(stmt string, isTerminal bool, modelDir string, db *sql.DB, ds string) error {
+func runStmt(stmt string, isTerminal bool, modelDir string, ds string) error {
```

Comment:
ds -> `datasource`?

Reply:
Good idea, will polish it in the next PR.